### ISSUE TITLE
Only a project team leader can see unassigned task in their unassing task list

### ISSUE THREAD

**Issue by Mailhot** [Rory-Sullivan/lambda#4]
At the moment only team leaders can view unassigned task (in their unassigned task list)
But a member can go into the project page and select view tasks from there to show all tasks (assigned or unassigned)
it would be more convenient to show them in their unassigned list also, so they could pick an unassigned task when they finished one. They have access to the information, it is just not displayed there.

**Comment by Rory-Sullivan** [comment]
Yeah I think you're right on this one. When I was testing this application I was mostly looking at it from a team lead point of view so I never really thought about it but it makes sense.
